bitnami/thanos/Chart.yaml

```diff
@@ -28,4 +28,4 @@ name: thanos
 sources:
   - https://github.com/bitnami/containers/tree/main/bitnami/thanos
   - https://thanos.io
-version: 12.5.1
+version: 12.6.0
```

Comment:
As I am editing an existing variable, and not adding a new one, is this a minor release or a major one?